### ISSUE TITLE
bug: Form elements do not have associated labels

### ISSUE THREAD

**Issue by Pareesh** [hlxsites/mammotome#509]
Hi, one of the reasons we are getting the low accessibility on page is that few form elements does not have associated labels with them. 

https://www.mammotome.com/us/en/about-us/investigator-initiated-study-request 
Ex: Field like lastname, city, state. 

Thank you
